server/taxonomy/taxonomyImportHelper.js

```diff
@@ -0,0 +1,63 @@
+const R = require('ramda')
```

Comment:
this file should probably be a TaxonomyImportService?

Reply:
no, it's used internally by the TaxonomyImportJob and CollectTaxonomiesImportJob